proxy.adoc

```diff
@@ -157,6 +158,36 @@ using the Zimbra Mailbox Ports.
 
 |====================================================
 
+=== Strict Server Name Enforcement
+
+Zimbra Proxy has the ability to strictly enforce which values are allowed in the `Host` header
+passed in by the client.
+
+This is _enabled by default_ on new installations but left _disabled for upgrades_ from previous
```

Comment:
nit: the use of italics seems inconsistent to me, how about:

`This is _enabled by default_ on *new installations* but left _disabled_ for *upgrades* from previous...`

This provides extra emphasis on the conditions used to determine when it is enabled (or not).
- **new installation[s]** ∴ _enabled (by default)_
- **upgrade[s]** ∴ _disabled_

Reply:
Please don't hesitate mentioning something if it makes the documentation better by any means!
I went ahead and made this last suggestion and pushed it into 'develop' since the rest was already merged.